diff -urN kiax/gui/Kiax2Application.cpp kiax-patched/gui/Kiax2Application.cpp
--- kiax/gui/Kiax2Application.cpp	2008-12-28 19:03:02.000000000 +1000
+++ kiax-patched/gui/Kiax2Application.cpp	2014-03-07 15:18:14.000000000 +1100
@@ -81,7 +81,7 @@
 	QLocalSocket *localSocket = localServer->nextPendingConnection();
 	if (!localSocket->waitForReadyRead(timeout))
 	{
-		qDebug(localSocket->errorString().toLatin1());
+		qDebug() << localSocket->errorString().toLatin1();
 		return;
 	}
 	QByteArray byteArray = localSocket->readAll();
@@ -105,13 +105,13 @@
 	localSocket.connectToServer(_uniqueKey, QIODevice::WriteOnly);
 	if (!localSocket.waitForConnected(timeout))
 	{
-		qDebug(localSocket.errorString().toLatin1());
+		qDebug() << localSocket.errorString().toLatin1();
 		return false;
 	}
 	localSocket.write(message.toUtf8());
 	if (!localSocket.waitForBytesWritten(timeout))
 	{
-		qDebug(localSocket.errorString().toLatin1());
+		qDebug() << localSocket.errorString().toLatin1();
 		return false;
 	}
 	localSocket.disconnectFromServer();
